administrator/components/com_content/models/article.php

```diff
@@ -324,6 +324,7 @@ public function getItem($pk = null)
 		if (!empty($menuItems))
 		{
 			$item->menulink = array(
+			    'menuid' => $menuItems[0]->id,
```

Comment:
See alignment of code is not looks good in this diff. This is not cs error but just looks odd.